source/extensions/filters/common/ext_authz/ext_authz_grpc_impl.cc

```diff
@@ -114,7 +114,9 @@ void GrpcClientImpl::onSuccess(std::unique_ptr<envoy::service::auth::v3::CheckRe
       const auto& ok_response = response->ok_response();
       copyOkResponseMutations(authz_response, ok_response);
     }
-  } else {
+  } else if (response->status().code() == Grpc::Status::WellKnownGrpcStatus::PermissionDenied ||
```

Comment:
Please add a runtime guard for this behavior change.

Reply:
I added a runtime guard and a changelog entry. Can you please take a look?